tests/pygeoapi-test-config-ogr.yml

```diff
@@ -0,0 +1,357 @@
+# =================================================================
```

Comment:
Same comment as for the `cite.config.yml`; could we read this file from [here](https://github.com/geopython/pygeoapi/blob/master/tests/pygeoapi-test-config-ogr.yml) instead?

And read `docker.config.yml` from [here](https://github.com/byteroad/ogcapi-simple/blob/master/pygeoapi/docker.config.yml)?